examples/discover_customization_examples/public/plugin.tsx

```diff
@@ -401,6 +402,28 @@ export class DiscoverCustomizationExamplesPlugin implements Plugin {
           },
         });
 
+        customizations.set({
+          id: 'flyout',
+          size: '60%',
+          title: 'Example custom flyout',
+          actions: {
+            getActionItems: () =>
+              Array.from({ length: 5 }, (_, i) => {
```

Comment:
You can update this number of extra items (2 default ones are always present) during testing to check how it works.